packages/plugin/package.json

```diff
@@ -37,10 +37,10 @@
   "devDependencies": {
     "@types/eslint": "7.28.0",
     "@types/graphql-depth-limit": "1.1.2",
+    "@types/lodash.lowercase": "^4.3.6",
     "bob-the-bundler": "1.5.1",
     "graphql": "15.5.3",
-    "typescript": "4.4.2",
-    "@types/lodash.camelcase": "4.3.6"
```

Comment:
my bad, we need to have `@types/lodash.lowercase` and not `@types/lodash.camelcase`